src/components/Drawer/Drawer.styles.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+import constants from "../../theme/constants";
+import spacing from "../../theme/spacing";
+import getThemeValue from "../../utils/getThemeValue";
+
+export const DrawerContainer = styled.div`
+  background-color: ${getThemeValue("onyx", "base")};
+  color: ${getThemeValue("white", "base")};
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  visibility: hidden;
+  transform: translateY(-100%);
+  opacity: 0;
+  transition: opacity 300ms ${constants.easing.easeInOutQuad};
```

Comment:
For transition timing, I used seconds in my event list row animation PR. Not sure if we want to standardize on ms/s